src/Psy/ExecutionLoop/ForkingLoop.php

```diff
@@ -166,6 +166,8 @@ private function serializeReturn(array $return)
                 $serializable[$key] = $value;
             } catch (\Exception $e) {
                 // we'll just ignore this one...
+            } catch (\Error $e) {
```

Comment:
Good catch! You've still got to catch `\Exception` for PHP 5.x, though, so it doesn't make a huge difference.